packages/bento-design-system/src/DateField/Calendar.tsx

```diff
@@ -99,8 +99,10 @@ export function Calendar(props: Props) {
           {weekdayLabels.map((d, index) => (
             <Box
               className={weekDay}
-              width={config.dayWidth}
-              height={config.dayHeight}
+              style={{
+                width: config.dayWidth,
+                height: config.dayHeight,
```

Comment:
We're no longer limiting the day size in config to be part of the spacing scale.